core/storage/collection/gae_models.py

```diff
@@ -69,7 +69,18 @@ def get_collection_count(cls):
         return cls.get_all().count()
 
     def commit(self, committer_id, commit_message, commit_cmds):
-        """Updates the collection using the properties dict, then saves it."""
+        """Updates the collection model with commit change, then saves it.
```

Comment:
Not sure what "commit change" means. Suggest: "Updates the collection model by applying the given commit_cmds, then saves it." (or similar).